easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2015b-32bitIDX.eb

```diff
@@ -0,0 +1,24 @@
+name = 'METIS'
+version = '5.1.0'
+#default 32-bit IDTYPEWIDTH, no patch used
+versionsuffix = '-32bitIDX'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/metis/overview'
+description = """METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
+and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the
+multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
+
+builddependencies = [('CMake', '3.3.2')]
+
+configopts = [' ', 'shared=1 ']
```

Comment:
@boegel I can put this in the 64 bit pionter `METIS` as well, but none of the software required until now. the 32 bit pointer `METIS` is only required by `OpenFOAM`.